octree/include/pcl/octree/impl/octree_search.hpp

```diff
@@ -80,10 +80,7 @@ OctreePointCloudSearch<PointT, LeafContainerT, BranchContainerT>::voxelSearch(
 template <typename PointT, typename LeafContainerT, typename BranchContainerT>
 int
 OctreePointCloudSearch<PointT, LeafContainerT, BranchContainerT>::nearestKSearch(
-    const PointT& p_q,
-    int k,
-    std::vector<int>& k_indices,
-    std::vector<float>& k_sqr_distances)
+    const PointT& p_q, int k, Indices& k_indices, std::vector<float>& k_sqr_distances)
```

Comment:
Not actionable: k should likely be made uindex_t in a future pass

Reply:
`k` is currently signed. Should it be make `index_t` instead?